### ISSUE TITLE
Cannot select zero as a configuration value in UI, but can manually type in and send a zero

### ISSUE THREAD

**Issue by TheKorn2** [AlCalzone/node-zwave-js#6647]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Snap

### Z-Wave JS UI version

9.7.1.04fdaf5

### ZwaveJS version

12.4.1

### Describe the bug

When using the up/down arrows on a configuration value, can select any number from 1-65535.  Should be able to select any number from 0-65535.  (top bound is untested/unverified, bug is about lower bound.)

### To Reproduce

1. Include Zooz zen14
2. Open configuration v4
3. select option 2, auto-turn-off timer duration (outlet 1)
4. see value is zero
5. hit up, option is now 1 (though unsent)
6. hit down, option **will not** decrement back to 0
7. click on text box, type in 0
8. hit send button, receive confirmation that zero was sent and set

### Expected behavior

Zero should be selectable if it's a valid value (as it is in the case of a zen14).

### Additional context

Whoops, off by 1!

**Comment by robertsLando** [comment]
I have a feel the problem is in the device configuration that is reporting a wrong min/max value? 
Could you send me the link to configuration db you find on the top of the expanded raw when you click on a node in the control panel table?

cc @blhoward2

**Comment by apella12** [comment]
I think the problem is in the zooz_template
```
	// Auto-On/Off-Timers for other series with fixed time unit
	"auto_timer_duration_base": {
		"valueSize": 4,
		"unit": "minutes",
		"minValue": 1,
		"maxValue": 65535,
		"defaultValue": 60,
		"unsigned": true
```
There are other similar templates with minValue=0, but seem to be with different firmwares
i.e.
```
	// Auto-On/Off-Timers for ZEN3x and ZEN7x
	"auto_timer_base_0x_3x_7x": {
		"valueSize": 4,
		"unit": "minutes",
		"minValue": 0,
		"maxValue": 65535,
		"defaultValue": 0,
		"unsigned": true,
		"options": [
			{
				"label": "Disable",
				"value": 0
			}
		]
	},
```
The Zen14 is using the first one.
```
		{
			"#": "2",
			"$import": "templates/zooz_template.json#auto_off_timer_duration",
			"label": "Auto Turn-Off Timer Duration (Outlet 1)"
		},
```
and 
```
	"auto_off_timer_duration": {
		"label": "Auto Turn-Off Timer Duration",
		"$import": "#auto_timer_duration_base"
	},
```

**Comment by robertsLando** [comment]
Could you submit a PR to zwave-js to fix this?

**Comment by TheKorn2** [comment]
> I have a feel the problem is in the device configuration that is reporting a wrong min/max value? Could you send me the link to configuration db you find on the top of the expanded raw when you click on a node in the control panel table?

Be happy to, but I'm not super well versed in zwave-js-ui. What are you asking me to do, if you still need it? (eli5)

**Comment by apella12** [comment]
@TheKorn2 
I can do this later today.  It is a zwave-js PR, not UI. Only caution is the configs for zwave-js are backed up due to parental leave

**Comment by TheKorn2** [comment]
Thanks! Since I can force the value with keyboard input it's not a burning issue for me, just one I figured should be addressed.

**Comment by apella12** [comment]
@TheKorn2 
Can you confirm the fingerprint?
```
			"productType": "0x7000",
			"productId": "0xb003",
```
@robertsLando 
Can you move to the zwave-js issues as config?

**Comment by TheKorn2** [comment]
@apella12 Yup.

 "manufacturerId": 634,

  "productId": 45059,

  "productType": 28672,
